modules/cli/src/main/scala/scala/cli/internal/ProcUtil.scala

```diff
@@ -43,4 +47,24 @@ object ProcUtil {
     new String(data, StandardCharsets.UTF_8)
   }
 
+  def interruptProcess(process: Process, logger: Logger): Unit = {
+    val pid = process.pid()
+    try
+      if (process.isAlive) {
+        logger.debug(s"Sending a SIGINT signal to process PID:$pid")
```

Comment:
Shouldn't this be a standard message?
```suggestion
        logger.debug("Interrupting running process")
```